content/en/docs/reference/config/config-status/index.md

```diff
@@ -0,0 +1,105 @@
+
+Istio 1.6 and later provides information about the propagation of configuration
+changes through the mesh, using the `status` field of the resource.
+Status is disabled by default, and can be enabled during install with:
+
+{{< text bash >}}
+$ istioctl manifest apply --set values.pilot.env.PILOT_ENABLE_STATUS=true --set values.global.istiod.enableAnalysis=true
+{{< /text >}}
+
+The `status` field contains the state of a resource's configuration with various
+informational messages, including:
+
+* The resource's readiness.
+* How many data plane instances are associated with it.
+* Information for the output of tools, such as `istioctl analyze`.
+
+For example, the `kubectl wait` command monitors the `status` field to determine
+whether to unblock configuration and resume. For more information, see
+[Wait for Resource Status to Apply Configuration](/docs/ops/configuration/mesh/config-resource-ready/).
+
+## View the `status` field
+
+To view the contents of the `status` field, use the following command:
+
+{{< text bash >}}
+$ kubectl get virtualservice <service-name> -o yaml
+{{< /text >}}
+
+In the output, the `status` field contains several nested fields with details
+about the process of propagating configuration changes through the mesh.
+when Istio applied a configuration successfully.
+
+{{< text yaml >}}
+status:
+  conditions:
+  - lastProbeTime: null
+    lastTransitionTime: "2019-12-26T22:06:34Z"
+    message: "61/122 complete"
+    reason: "stillPropagating"
+    status: "False"
+    type: Reconciled
+  - lastProbeTime: null
+    lastTransitionTime: "2019-12-26T22:06:56Z"
+    message: "1 Error and 1 Warning found. See validationMessages field for details"
+    reason: "errorsFound"
+    status: "False"
+    type: PassedAnalysis
+  validationMessages:
+  - code: IST0101
+    level: Error
+    message: 'Referenced gateway not found: "bogus-gateway"'
+  - code: IST0102
+    level: Warn
+    message: 'mTLS not enabled for virtual service'
+{{< /text >}}
+
+## The `conditions` field
+
+The `type` field indicates the configuration condition. It can have the
+following values:
+
+* `PassedAnalysis`
+* `Reconciled`
+
+When you apply a configuration, a condition of type `Reconciled` and `status`
+equals `False` is added to the `conditions` field. The status becomes `True`
+when the resource finishes reconciling. The `status` field might transition to
+`True` instantaneously, depending on the speed of the cluster.
+
+The condition of type `PassedAnalysis` has a status of `True` or
+`False` to indicate the outcome of configuration validation. `PassedAnalysis` is
+an informational field only, meaning that it does not block the application of
+an invalid configuration. For example, it is possible for the status to indicate
+that validation failed, but applying the configuration was successful. This
+means Istio was able to set the new configuration, but the configuration was
+invalid, likely due to a syntax error or similar problem.
```

Comment:
1. This section still doesn't explain what "reconciling" actually means and refers to `The condition of type ParsedAnalysis`, but doesn't explain when it was added.

2. Re: the sentence that says `PassedAnalysis is an informational field ...`, PassedAnalysis is not a field, it's a condition type.

3. This also should mention that if validation fails the reason is in the `validationMessages` field.

Any reason why you didn't accept my suggestion in https://github.com/istio/istio.io/pull/7157#discussion_r419467492?

Reply:
Committed via the GitHub UI but was missing after the merge fix, so I restored it.

Reply:
This suggestion also seems to have gotten lost: https://github.com/istio/istio.io/pull/7157#discussion_r418208921

Reply:
Also lost: https://github.com/istio/istio.io/pull/7157#discussion_r419472240

Reply:
Maybe other suggestions from @therealmitchconnors are missing?

Reply:
Added new commits to restore what seems to be changes made via the GitHub UI.